package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "iitc-ce",
+  "description": "ingress intel total conversion (IITC-CE)",
+  "keywords": [
+    "iitc",
+    "ingress"
+  ],
+  "main": "core/total-conversion-build.js",
+  "repository": "https://github.com/IITC-CE/ingress-intel-total-conversion.git",
+  "author": ".",
+  "license": "ISC",
+  "private": true,
+  "devDependencies": {
+    "eslint": "^7.16.0"
```

Comment:
https://eslint.org/blog/2021/02/eslint-v7.20.0-released

Reply:
Thats the job of the package manager. There is no need to keep track of every update.
`npm install`/`yarn install` (or upgrade) will handle it by themself.